### PR TITLE
Supports day format in ReadableDuration

### PR DESCRIPTION
**Pull request by killme2008**
## What have you changed? (mandatory)

Make `util::config::ReadableDuration` supports day format for `Serialize/Deserialize`, such as `1d2h` that means one day and two hours.

## What are the type of the changes? (mandatory)

It's  a improvement.It make `ReadableDuration` more flexible for configuration.

## How has this PR been tested? (mandatory)

I modified the `test_parse_readable_duration` test case in `util::config` module, and adds new assertions for parsing the day format, including only days format and parsing days with other time units.

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

No.

## Does this PR affect tidb-ansible update? (mandatory)

No.

### CODE REVIEW

File: src/util/config.rs [hunk 1]
```diff
@@ -518,7 +525,7 @@ impl<'de> Deserialize<'de> for ReadableDuration {
                     if unit >= last_unit {
                         return Err(E::invalid_value(
                             Unexpected::Str(dur_str),
-                            &"h, m, s, ms should occur in giving order.",
+                            &"d, h, m, s, ms should occur in giving order.",
```
Comment (ice1000) [inline src/util/config.rs]: I consider it a legacy typo. Should we change "giving" into "given"? @tikv/pingcap-i18n

**Comment by rleungx** [comment]
@killme2008 Thanks for your contribution! Please sign-off the DCO.

File: src/util/config.rs (hunk 1 above)
Comment (lilin90) [reply src/util/config.rs]: @ice1000 Yes. I think we should change it to "given".

Comment (killme2008) [reply src/util/config.rs]: May i change it in this PR? @lilin90 @ice1000

**Comment by killme2008** [comment]
@rleungx  OK, i do it right now. Thanks.

File: src/util/config.rs (hunk 1 above)
Comment (ice1000) [reply src/util/config.rs]: I think it's fine. Do it.

Comment (ice1000) [reply src/util/config.rs]: I think it's fine. Ping @lilin90 @overvenus

**Comment by killme2008** [comment]
@overvenus  ok

**Comment by killme2008** [comment]
@overvenus  Fixed it at https://github.com/tikv/tikv/pull/3687/commits/431c5e99bb95f1e3a7b11b5655954688102c8fa8

**Comment by overvenus** [comment]
Good job! could you sign off your commits?

**Pull request merged**